data/volunteers.json

```diff
@@ -1,22 +1,168 @@
 {
 	"blurb": {
-		"description": "#",
+		"description": "UXPH is always looking for passionate and enthusiastic individuals who want to be a part of the growing Filipino design community! Whether you're a professional, student, or simply a design enthusiast, we're happy to have you join us! Check out our openings below.",
 		"img_url": "#"
 	},
 
 	"perks": {
-		"perk_name": {
-			"name": "<name>",
-			"description": "<description>",
+		"perk1": {
+			"name": "Access to the largest network of UX designers in the Philippines",
+			"description": "Just by joining UXPH you already gain access to an invaluable network of designers to connect, share, and collaborate with",
+			"icon": "<icon>"
```

Comment:
Look for appropriate icons already in font-awesome.
Write something like `fab fa-twitter` here. Do this for the rest.